test/acceptance.go

```diff
@@ -86,7 +86,7 @@ func NewAcceptanceTest(t *testing.T, opts *AcceptanceOpts) *AcceptanceTest {
 func freeAddress() string {
 	// Let the OS allocate a free address, close it and hope
 	// it is still free when starting Alertmanager.
-	l, err := net.Listen("tcp", ":0")
+	l, err := net.Listen("tcp4", ":0")
```

Comment:
Actually, the normal stdlib `ListenAndServe()` function, which is essentially used everywhere also uses [this](https://golang.org/src/net/http/server.go#L1868).

Accordingly none of our tools serving HTTP content should work for you then – confused.